cmd/uhc/cluster/list/list.go

```diff
@@ -131,7 +184,21 @@ func init() {
 		&args.step,
 		"step",
 		false,
-		"load pages one step at a time",
+		"Load pages one step at a time",
+	)
+	flags.StringVar(
+		&args.columns,
+		"columns",
+		"",
```

Comment:
It may be better to set a default value for this option. Then you don't need to check if the option has been provided.

Reply:
default:
id, name, api.url, version.id, region.id